src/test/java/com/commerce/backendserver/member/domain/MemberQueryRepositoryTest.java

```diff
@@ -0,0 +1,55 @@
+package com.commerce.backendserver.member.domain;
+
+import com.commerce.backendserver.common.base.RepositoryTestBase;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Optional;
+
+import static com.commerce.backendserver.common.fixture.MemberFixture.A;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("[MemberQueryRepository Test] (Domain layer)")
+public class MemberQueryRepositoryTest extends RepositoryTestBase {
```

Comment:
```suggestion
class MemberQueryRepositoryTest extends RepositoryTestBase {
```

Reply:
> JUnit5 is more tolerant regarding the visibilities of Test classes than JUnit4, which required everything to be public.
> 
> In this context, JUnit5 test classes can have any visibility but private, however, it is recommended to use the default package visibility, which improves readability of code.

[[SonarCloud] Remove this 'public' modifier.](https://sonarcloud.io/project/issues?resolved=false&severities=INFO&types=CODE_SMELL&id=corona-warn-app_cwa-server&open=AX0OqBCAjN8jugUjR2bF&tab=why)